js/effects.js

```diff
@@ -0,0 +1,128 @@
+const picture = document.querySelector('.img-upload__preview');
+
+const EFFECTS = [
+  {
+    name: 'none',
+    style: 'none',
+    min: 0,
+    max: 100,
+    step: 1,
+    unit: '',
+  },
+  {
+    name: 'chrome',
+    style: 'grayscale',
+    min: 0,
+    max: 1,
+    step: 0.1,
+    unit: '',
+  },
+  {
+    name: 'sepia',
+    style: 'sepia',
+    min: 0,
+    max: 1,
+    step: 0.1,
+    unit: '',
+  },
+  {
+    name: 'marvin',
+    style: 'invert',
+    min: 0,
+    max: 100,
+    step: 1,
+    unit: '%',
+  },
+  {
+    name: 'phobos',
+    style: 'blur',
+    min: 0,
+    max: 3,
+    step: 0.1,
+    unit: 'px',
+  },
+  {
+    name: 'heat',
+    style: 'brightness',
+    min: 1,
+    max: 3,
+    step: 0.1,
+    unit: '',
+  }
+];
+const DEFAULT_EFFECTS_VALUE = 100;
+const DEFAULT_EFFECT = EFFECTS[0];
+let chosenEffect = DEFAULT_EFFECT;
+
+const sliderContainer = document.querySelector('.img-upload__effect-level');
+const sliderElement = document.querySelector('.effect-level__slider');
+const effectsElement = document.querySelector('.effects');
+const effectLevelElement = document.querySelector('.effect-level__value');
+
+const isDeffault = () => chosenEffect === DEFAULT_EFFECT;
```

Comment:
```suggestion
const isDefault = () => chosenEffect === DEFAULT_EFFECT;
```

Есть расширения для VS code которые ловят опечатки
[вот например ](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)